java/java.source.base/src/org/netbeans/modules/java/source/indexing/VanillaCompileWorker.java

```diff
@@ -712,7 +712,7 @@ public Void visitClass(ClassTree node, Void p) {
                             csym.members_field.remove(member);
                         }
                     }
-                    if (errorClass || def.hasTag(JCTree.Tag.ERRONEOUS) || def.hasTag(JCTree.Tag.BLOCK)) {
```

Comment:
From the logs I see you wrote the original  JCTree.Tag.BLOCK condition here, so I'm sure the adjustment makes sense.